course_content/notebooks/matplotlib_intro.ipynb

```diff
@@ -374,20 +419,20 @@
L7yVBqAET54kf7sBfX448J1F3RDQ3RUNkrW4a4\ncCBTYwKZjocUF3d5LOxKrbDz8qy46qooRETIIxo6p3ymO3D+miAWYj7aoj06otOf/t/16Ir92AcD\ngaUwxIUD2YqXJiRE76WpV69apeKw09KKlRa21SZaq1ZTDLEtD4d9BqfRGE3KTMFsimZIw/mAg8f1\nw9UT3XUprQYNKhctkpFRrOSwdfaez1gKtE17JjJRghK0/L/2zjs+y+r8/+8AIiCrTAHBgYqgTGWH\nISiIWCq1altX1VI7bdVaR/VXLQiCiFVrXYWKgrhABNkrzIS9R1hBZhaE7PXk+fz+OIngyDnnyQDM\n93m/XnmRhOu5z33nvs91X+dah8u//l0EEdzCEI5xlPU4tm4vhsaFFrYrJbFRPfcK1/iw/48rbB93\nCPgFHUujsFeyggwyGcDA7/xfE5pSlfM5wIESHbuRR+AD/NwiTZqcez5sl8JOLlwduSZzaRS2rZT5\nfM6nMRdyKITd1U+nSaFLxIavS+tcUtgZGXlIUKuW3d3gM/cSA2YlWVI2sp6OdPxOp8WqVOVOfslC\n5nOMoyEft0Ylk+p70hU09rawy3/undMK++jRdOeSDCDhODRytKI9UsIJn0UWUSzmDn7+vRZaBBFc\ny7UsI8p7h+fTaVDZNFF3VWA2qm/arNpo0qQWhw+nhXwO5cXBg6nO7coSjhsLxoYEh/OhWQmCjgUU\nsJtdtLRsH3UZLdnJjtAPjjmnQw4L2+feAVx0UW2++srdjvZMcPhwGk2a1HTuXpSQ7J57R0thLJ3k\nJJvZREc6fe//N6QhNzKAOcwu0fGbVIEjrvtXz33/igL+wZLsYxcC57TC3rQpnnbt7O1L8/Nhz1fQ\n6hL7sTbmQAePFqXfZi1ruJprrEHFznQlgghm82XI/rTKEWYT0K2O5vxtWsL2vXaZK6+sz9Gj6Zw8\n6ZE4Ws4Eg2LNmiNcd52lyTWwbQ+0dhSu7c4zW4TVK4GVtpoYfkQ9mtKsWJmudGMrW0jCsTHo93Bt\nNVjn+HM3bWQKMJIdVnbnzk1ZvfpIyOdQHqxff5SOHZs45bbtNc+mjc050LYEcy+bbD5gIr3pa93N\nvj0dOU4yCYQe9OtQzegGG60vc8+9mjWrcuGFNb22NCwN57TCXrv2KJ072yf8rjho0cQE5myszoYu\nIbarDRBgDTHF9lEuojKVuZNfcIiDLGdZaINgJv16x0PT4SrYHGuXqVKlEtde25S1a8/+pN+xI4n6\n9Ws4V0ibY6F9K/uxVmVDjxK0Gs4gg2VEcTO3WHtP1KQmvelbohdum/ONSyvZ4haJiIB2V8JmR8p+\nZGQLVqwomWumrImJOUy3bsW/5MAEwrfudt+/9TnmGQ+FAAGmMJmWXO6cf1WoQhe6sYqVoQ2C0Qmr\nHcH8Dq1hk0e5RefOzcp97p3TCnvduqN07mx/aDbtMsrMRnzAbHbbMsQl9Va20IjGNOZCp2w1qnE3\n97KWNexlT0jjXFcd1rkemqtg0073sbp1a0ZMjGPPpjPAihUHiYx07KtE4f1rbZdZkw1dS6CwFzCP\nDnSiIe59LrvSjTTSQnaNVI6AztXdqZntW7kn/VVXNSAtLZcjR86+Wysm5gjdutn3Ndt/yOQo/8jS\n/0wyz/Z1Id6/ecylBjW4yXOvxs50YRc7SSe0v10Xj3vXsrnxYZ90HLpz56asXRu6Lz0USquwDwBb\ngI3AmlKfzWkcO5ZOamoOl19ud3Bu9lDYawut61Ci1EKsYgU9ifT+TG3qcD39WMNq/4Hws7CvuhQO\nHHWn9nXrdhExMWffwl6+/CCRkfbcZsnPwt6QDZ1CnPCHOMhe9tCX673kK1OZwdzCXOaEnDHS0WNZ\n3aG1e4UUERFBz55n38rOzs5nx44kOnWyu0R87t3RgGlB69ro9nTyyGMzG7mFId472degBm1px2pi\n/AcCOlWD7bn2GFLlytCulfv+de3ajFWrHJvLlpLSKmwBfYGOQJdSn81pLFlygL59L3EWEfhY2CVx\nh+xjL4JvpBL50IarOUBcSCXP7avBrlx7+9CqVY2ffpvDeO/a9SJiYg47W5qWNytWHKRXr4utMgeP\nQvXz7UGrgGBbLrQPYS/AIEFm8SU3MpBq+K/FL6MlzWjGihDdWp2qwQaHwm7vMeEBevU6+wp7w4Zj\ntGnT0Nk9cONO99xbl2Os61CMpZ3soDktqIk74eB0utOTdawlD/+eLDUqmY14N3m4JF0rpC5dmhEb\ne5yUFIdVVQrKwiVSLmVZixfH0a+ffXNbyU9hrymBwl7JCnrQM+S+u9WoxhVcyTa2en+meiW4sqo7\n8NixNWxwrNibNq1FzZpV2bvXI/G3nDh4MJXs7HyuuMK+Otq0y22h7cqFi86DWiEEHDeynspUpj0d\n/D9UyEAGEUM0KXjk4RXiY2FffTnsPuDuShgZ2YLly8+uwvbxX4N5Fjs63Fnrs+G6EP3Xm9hIBzqG\n9iGgPvW5mIvZGGIhW5dqsMZHYTtckuefX4Xu3S9i2TJHWWspKAsLeyGwDhhW+tM5xaJFcfTvb1fY\nRxKgSmV7yWhQp1wiviQQTyIJtKO9/4dOowMd2cTGkD5zrYcfu2NrY9W46N79orPqxy7yX7tSwjbH\nul+2G3KMBetLDjksYiGDHYHG4qhLXbrTg/nM9f7MFVUhqQBSLFmd1avBZRfBzn32Y3Xq1IS9e0+Q\nmnr2Mn18/NdgnkWXwl6XY55tX9JI4yhHuArHgYuhB5FEszKkQjYfP7aPhQ3Qv/+lLFoU5z12qJRW\nYffEuEMGAX8Aep3+n88999zXX1FRUd4H3bo1gUAgyFVX2bcBitkM115tP9aWXGhQxb/xjBDzmEt3\nepZ4c4LLaEkGGRzE/03brTqsyLLLdG4L0R5NyiIjW5TrQ+Ni3rx99O17iVPO5/6tyjZBPV82s4
```

Comment:
I strongly dislike starting a sentence with a limited-usage noun like this. It would be better like the following, which moves the noun off of the start of the sentence:
```
A filled contour plot is created using **`contourf`**:
```

You can add bold emphasis to make the plot function you're introducing more obvious.